src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -26,6 +27,22 @@ export default function CustomFieldInputSingleChoice(props) {
     fill="skip"
   />);
 
+  const clear = () => {
+    setValue(undefined);
+    setSearchValue(undefined);
+  };
+
+  const clearIcon = () => {
+    if (!props.readOnly && (!!value || !!searchValue)) {
```

Comment:
Nit: I don't think we really use the `!!` pattern for boolean coercion anywhere, and I think it's a bit less readable than comparing to the values that we don't want. Also, do we actually need more than the default truthy check?

Reply:
Looks cleaner too :)